bms_sender_test.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import BatteryManagementSystem
+class test_bms_streaming(unittest.TestCase):
+    def test_incorrect_stream_format(self):
+        self.assertTrue(BatteryManagementSystem.getBMSStream(stream_length = 100, stream_format = "list") == 'UNSUPPORTED_STREAM_FORMAT')
+    def test_bms_stream_in_json_format(self):
+        BatteryManagementSystem.getBMSStream(stream_length = 100, stream_format = "json")
```

Comment:
No test for invalid stream format? The code is present... how did you check that it's working?